### PR TITLE
add Winthrop; refactor YP preprod permissions

### PR DESCRIPTION
**Pull request by karenhawkinson**
Asana tickets:
[Add New Youth Pass Partners - WINTHROP 9.2](https://app.asana.com/0/1170867711449497/1202798984953626/f)
[Replace Test MBTA Emails With @example.com Emails](https://app.asana.com/0/1170867711449497/1202553442176425/f)

This PR also removes Newton and Reading admin from the Training data source.

Note the emails in the PreProd data source now follow a consistent format: municipality-admin#@example.com, excepting the first Malden email which is a test account.

### CODE REVIEW

File: data_sources/PreProd Youth Permissions.csv [hunk 1]
```diff
@@ -1,165 +1,166 @@
...
+02150,Chelsea,chelsea-admin1@example.com,chelsea-admin2@example.com,chelsea-admin3@example.com,chelsea-admin4@example.com,,,,,,,notificationemail@example.com
+02149,Everett,everett-admin1@example.com,everett-admin2@example.com,everett-admin3@example.com,everett-admin4@example.com,,,,,,,notificationemail@example.com
+01701,Framingham,framingham-admin1@example.com,framingham-admin2@example.com,framingham-admin3@example.com,framingham-admin4@example.com,,,,,,,notificationemail@example.com
+01702,Framingham,framingham-admin1@example.com,framingham-admin2@example.com,framingham-admin3@example.com,framingham-admin4@example.com,,,,,,,notificationemail@example.com
+02420,Lexington,lexington-admin1@example.com,lexington-admin2@example.com,lexington-admin3@example.com,lexington-admin4@example.com,,,,,,,notificationemail@example.com
+02421,Lexington,lexington-admin1@example.com,lexington-admin2@example.com,lexington-admin3@example.com,lexington-admin4@example.com,,,,,,,notificationemail@example.com
+02148,Malden,maldentestuser@example.com,malden-admin2@example.com,malden-admin3@example.com,malden-admin4@example.com,,,,,,,notificationemail@example.com
```
Comment (krisrjohnson21) [inline data_sources/PreProd Youth Permissions.csv:87]: We can change the maldentestuser email to follow the same naming convention. We no longer use the Malden automation account due to problems with two-factor login

**Review by krisrjohnson21 — APPROVED** [review]
Looks great! Approving with one small suggestion for Malden, not blocking ✅

File: data_sources/PreProd Youth Permissions.csv (hunk 1 above)
Comment (karenhawkinson) [reply data_sources/PreProd Youth Permissions.csv]: Great point. First Malden email now fits the naming convention.

**Review by krisrjohnson21 — APPROVED** [review]
✅

**Pull request merged**